File: cs50x/aula_python/phonebook.py
from cs50 import get_string

people = {
    "Brian": "+1-617-495-1000",
    "David": "+1-949-468-2750"
}

print("caraca!")

name = get_string("Name: ")
#name = input("Name: ")
if name in people:
    print(f"Number: {people[name]}")

File: cs50x/aula_c/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    string names[] = {"Carter", "David"};
    string numbers[] = {"+1-617-495-1000", "+1-949-468-2750"};

    string name = get_string("Name: ");
    for (int i = 0; i < 2; i++)
    {
        if (strcmp(names[i], name) == 0) //C não compara strings diretamente, por isso usamos essa função para comparar
        {
            printf("Found %s\n", numbers[i]);
            return 0;
        }
    }
    printf("Not found\n");
    return 1;
}